Source/f_finale.c

```diff
@@ -842,8 +842,12 @@ void F_Drawer (void)
     if (!finalestage || !gamemapinfo->endpic[0] || (strcmp(gamemapinfo->endpic, "-") == 0))
     {
       F_TextWrite();
-      return;
     }
+    else
```

Comment:
Does this deviate from the reference implementation?

Reply:
No, now it matches with the reference implementation:
https://github.com/coelckers/prboom-plus/blob/a43dff0255b6d81c5115010f27cd65b2598ded98/prboom2/src/f_finale2.c#L164-#L169

Reply:
See, thanks!